mk/main.mk

```diff
@@ -3,7 +3,7 @@ GO_LDFLAGS := -X `go list ./version`.GitCommit=`git rev-parse --short HEAD`
 GO_GCFLAGS :=
 
 # Full package list
-PKGS := $(shell go list -tags "$(BUILDTAGS)" ./commands/... ./libmachine/... ./drivers/... | grep -v "/vendor/" | grep -v "/Godeps/")
+PKGS := $(shell go list -tags "$(BUILDTAGS)" ./... | grep -v "/vendor/" | grep -v "/Godeps/" | grep -v "/cmd")
```

Comment:
The intent was to ignore `/cmd` since we need to do some cleanup with the drivers expatriation and right now we redefine main in the same package. Moving to blacklist meanwhile.